test/integration/dialog-popover.test.js

```diff
@@ -84,33 +84,6 @@ describe('popover in dialog', () => {
         expect(dialog.opened).to.be.true;
       });
     });
-
-    describe('modeless dialog', () => {
```

Comment:
These tests don't make sense anymore: even if `bringToFront()` sets higher z-index on the overlay, the popover will still be shown on top of it because it's using popover, and the dialog currently does not. 

I'll create a separate issue to add a new logic for bringing nested overlays to front and then re-add ITs.